scripts/cronned_results_scripts.sh

```diff
@@ -6,9 +6,6 @@ cd "$(dirname "$0")"/..
 # export url paramter.
 time (cd webroot/results/admin/; REQUEST_URI="export=" php export_public.php)
 
-# Update statistics page
-time (cd webroot/results/; php statistics.php update)
-
 # Update Evolution of Records
 # We need to set SERVER_NAME to avoid generating the "This is only a copy of
 # the WCA results system used for testing stuff" message.
```

Comment:
Shouldn't the entire lower part of this script also be deleted?
As far as I'm concerned, Evolution of Records and Missing Averages have been ported over as well and don't need to be recomputed.

Reply:
As such, the corresponding files under `webroot/results/misc` could also be deleted. I don't think we need the `evolution` and `missing_averages` folders there anymore either.